activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```diff
@@ -314,6 +314,20 @@ instances:
     #
     # tls_ca_cert: <CA_CERT_PATH>
 
+    ## @param tls_protocols_allowed - list of strings - optional
+    ## The expected versions of TLS/SSL when fetching intermediate certificates.
+    ## The supported versions and values are:
+    ##   SSLv3
+    ##   TLSv1
+    ##   TLSv1.1
```

Comment:
Are these versions supposed to be here? <1.2 is deprecated by OpenSSL

Reply:
They are supported but not by default. I'll rephrase because it's not clear, but this list is the list of possible values for this option, and only SSLv3, TLSv1.2, TLSv1.3 are by default